src/api.rs

```diff
@@ -482,5 +488,9 @@ pub mod reply {
 
         WriteCertificate:
           - id: CertId
+
+        // reply size is chosen to be smaller than the largest standard syscall, Encrypt
+        Extension:
+          - reply: Bytes<{ MAX_MESSAGE_LENGTH + 2 * MAX_SHORT_DATA_LENGTH }>
```

Comment:
Yes, that's the part I do not get. Why do you not want to serialize Encrypt, if you want to encrypt data using another backend?

Reply:
If a backend wants to implement the `Encrypt` syscall, it can just use the existing request implementation.  This variant is only relevant for API extensions, i. e. new syscalls that are only available in some backend.  For these syscalls, we use serialization to avoid adding a type parameter.  And to store the serialized data, we need a fixed buffer size.  It is chosen so that the enums do not grow larger than already needed, but the size could also be set to a different value.

Reply:
Ideally, the buffer size would be configured by the runner, but we can’t do that atm.

Reply:
Updated the comment.